cmd/cli/install.go

```diff
@@ -55,33 +56,27 @@ well as for adding a Kubernetes Namespace to the list of Namespaces a control
 plane should watch for sidecar injection of Envoy proxies.
 `
 const (
-	defaultCertificateManager            = "tresor"
-	defaultCertManagerIssuerGroup        = "cert-manager.io"
-	defaultCertManagerIssuerKind         = "Issuer"
-	defaultCertManagerIssuerName         = "osm-ca"
-	defaultChartPath                     = ""
-	defaultContainerRegistry             = "openservicemesh"
-	defaultContainerRegistrySecret       = ""
-	defaultMeshName                      = "osm"
-	defaultOsmImagePullPolicy            = "IfNotPresent"
-	defaultOsmImageTag                   = "v0.8.3"
-	defaultPrometheusRetentionTime       = constants.PrometheusDefaultRetentionTime
-	defaultVaultHost                     = ""
-	defaultVaultProtocol                 = "http"
-	defaultVaultToken                    = ""
-	defaultVaultRole                     = "openservicemesh"
-	defaultEnvoyLogLevel                 = "error"
-	defaultControllerLogLevel            = "info"
-	defaultServiceCertValidityDuration   = "24h"
-	defaultEnableDebugServer             = false
-	defaultEnableEgress                  = false
-	defaultEnablePermissiveTrafficPolicy = false
-	defaultDeployPrometheus              = false
-	defaultEnablePrometheusScraping      = true
-	defaultDeployGrafana                 = false
-	defaultEnableFluentbit               = false
-	defaultDeployJaeger                  = false
-	defaultEnforceSingleMesh             = false
+	defaultCertificateManager      = "tresor"
```

Comment:
Can the install cli changes be carved out into a separate PR? Since they only update charts, it doesn't matter to leave them here for a bit longer.

This is going to cause a lot of conflicts with https://github.com/openservicemesh/osm/pull/3255

Reply:
Yes.

we can keep the CI arguments in install.go. But they won't be added in osm-controller deployment

Reply:
install.go reverted